src/trigger-method.js

```diff
@@ -3,66 +3,73 @@
 // Trigger Method
 // --------------
 
-Marionette._triggerMethod = (function() {
-  // split the event name on the ":"
-  var splitter = /(^|:)(\w)/gi;
+import _ from 'underscore';
 
-  // take the event section ("section1:section2:section3")
-  // and turn it in to uppercase name
-  function getEventName(match, prefix, eventName) {
-    return eventName.toUpperCase();
-  }
+var exports = {
+  _triggerMethod: (function() {
```

Comment:
So is _triggerMethod. I'll just keep these in the exports for now, get it working, then we can think of what to do, if anything.